cookbook/how_to_install_extension_from_command_line.rst

```diff
@@ -51,8 +51,8 @@ Finish the installation by clearing the cache again:
 Using Composer
 ~~~~~~~~~~~~~~
 
-Start with upgrading Composer to the latest version. This may be needed in case the extension to be
-installed uses some bleedy edge feature in its ``composer.json`` file:
+Start with upgrading Composer to the latest version. This may be needed in case if the extension to be
```

Comment:
“bleedy edge” here should be bleeding edge.

From: Christian Flothmann [mailto:notifications@github.com]
Sent: Tuesday, August 04, 2015 1:23 PM
To: orocrm/documentation <documentation@noreply.github.com>
Subject: Re: [documentation] OD-117: Review Book and Cookbook (#160)


In cookbook/how_to_install_extension_from_command_line.rst<https://github.com/orocrm/documentation/pull/160#discussion_r36236594>:

> @@ -51,8 +51,8 @@ Finish the installation by clearing the cache again:

>  Using Composer

>  ~~~~~~~~~~~~~~

>

> -Start with upgrading Composer to the latest version. This may be needed in case the extension to be

> -installed uses some bleedy edge feature in its ``composer.json`` file:

> +Start with upgrading Composer to the latest version. This may be needed in case if the extension to be

I think the sentence was correct before without the "if".

—
Reply to this email directly or view it on GitHub<https://github.com/orocrm/documentation/pull/160/files#r36236594>.